src/sentry/static/sentry/app/components/group/releaseChart.jsx

```diff
@@ -165,13 +163,13 @@ const GroupReleaseChart = createReactClass({
           label={t('events')}
           markers={markers}
           barClasses={['release', 'environment', 'inactive']}
-          tooltip={this.renderTooltip}
+          tooltip={this.renderTooltip.bind(this)}
```

Comment:
I would prefer making `renderTooltip` a class property, that way we don't rebind and create a new function on every render.